server/views/record-of-oral/risk-assessment.njk

```diff
@@ -81,7 +92,11 @@
       id: "likelihoodAssessment",
       label: {
         text: "Your assessment"
-      }
+      },
+      value: data.likelihoodAssessment,
+      errorMessage: {
+        text: "Enter the tool name"
```

Comment:
Does this error message need to be "Enter your assessment"?

Reply:
Good catch.